warp10/src/main/sh/warp10-standalone.sh

```diff
@@ -432,11 +432,13 @@ start() {
     fi
   fi
 
-  if [ "$(find -L ${LEVELDB_HOME} -maxdepth 1 -type f | wc -l)" -eq 0 ]; then
-    echo "Init leveldb"
-    # Create leveldb database
-    echo \"Init leveldb database...\" >> ${WARP10_HOME}/logs/warp10.log
-    ${JAVACMD} ${JAVA_OPTS} -cp ${WARP10_CP} ${WARP10_INIT} ${LEVELDB_HOME} >> ${WARP10_HOME}/logs/warp10.log 2>&1
+  if [ "${IN_MEMORY:-}" != "true" ]; then
```

Comment:
You can also include the creation of the directory above in the if block.

Reply:
done